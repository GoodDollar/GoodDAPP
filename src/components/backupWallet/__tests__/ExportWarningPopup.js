import React from 'react'
import renderer from 'react-test-renderer'
import { withThemeProvider } from '../../../__tests__/__util__'
import ImportedWarningPopup from '../ExportWarningPopup'

const ExportWarningPopup = withThemeProvider(ImportedWarningPopup)

describe('ExportWarningPopup', () => {
  it('renders without errors', () => {
    let tree

    expect(() => (tree = renderer.create(<ExportWarningPopup />))).not.toThrow()
    expect(tree.toJSON()).toBeTruthy()
  })

  it('matches snapshot', () => {
    const tree = renderer.create(<ExportWarningPopup />)

    expect(tree.toJSON()).toMatchSnapshot()
  })
})
