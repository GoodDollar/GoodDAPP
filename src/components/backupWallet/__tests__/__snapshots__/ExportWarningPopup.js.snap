// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExportWarningPopup matches snapshot 1`] = `
<div
  className="css-view-1dbjc4n"
  style={
    Object {
      "WebkitBoxFlex": 1,
      "WebkitFlexBasis": "0%",
      "WebkitFlexGrow": 1,
      "WebkitFlexShrink": 1,
      "flexBasis": "0%",
      "flexGrow": 1,
      "flexShrink": 1,
      "msFlexNegative": 1,
      "msFlexPositive": 1,
      "msFlexPreferredSize": "0%",
    }
  }
>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "WebkitBoxPack": "justify",
        "WebkitJustifyContent": "space-around",
        "display": "flex",
        "height": "100%",
        "justifyContent": "space-around",
        "marginBottom": "auto",
        "marginTop": "auto",
        "maxHeight": "473.76623376623377px",
        "minHeight": "310px",
        "msFlexPack": "distribute",
        "width": "389.2361111111111px",
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "center",
            "WebkitBoxAlign": "center",
            "alignItems": "center",
            "display": "flex",
            "height": "124.67532467532467px",
            "marginBottom": "19.948051948051948px",
            "marginTop": "8px",
            "msFlexAlign": "center",
            "width": "100%",
          }
        }
      >
        <div
          className="css-text-901oao"
          dir="auto"
          style={
            Object {
              "WebkitTransform": "rotate(180deg)",
              "color": "rgba(0,175,255,1.00)",
              "fontFamily": "gooddollar",
              "fontSize": "100px",
              "fontStyle": "normal",
              "fontWeight": "normal",
              "transform": "rotate(180deg)",
            }
          }
        >
          î ž
        </div>
      </div>
    </div>
    <div
      className="css-text-901oao"
      dir="auto"
      style={
        Object {
          "color": "rgba(250,108,119,1.00)",
          "direction": "ltr",
          "fontFamily": "Roboto Slab",
          "fontSize": "22px",
          "fontWeight": "700",
          "lineHeight": "29px",
          "marginBottom": "8px",
          "textAlign": "center",
          "textDecoration": "none",
          "textTransform": "none",
        }
      }
    >
      Do Not Send Tokens
From Ethereum Network
To This Address
    </div>
    <div
      className="css-text-901oao"
      dir="auto"
      style={
        Object {
          "color": "rgba(66,69,74,1.00)",
          "direction": "ltr",
          "fontFamily": "Roboto",
          "fontSize": "16px",
          "fontWeight": "normal",
          "lineHeight": "22px",
          "textAlign": "center",
          "textDecoration": "none",
          "textTransform": "none",
        }
      }
    >
      Keep in mind - This is an internal
network address for G$ tokens only.
    </div>
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitBoxDirection": "normal",
          "WebkitBoxOrient": "horizontal",
          "WebkitBoxPack": "end",
          "WebkitFlexDirection": "row",
          "WebkitJustifyContent": "flex-end",
          "display": "flex",
          "flexDirection": "row",
          "justifyContent": "flex-end",
          "msFlexDirection": "row",
          "msFlexPack": "end",
          "paddingLeft": "0px",
          "paddingRight": "0px",
          "paddingTop": "19.948051948051948px",
        }
      }
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "stretch",
            "WebkitBoxAlign": "stretch",
            "WebkitBoxPack": "center",
            "WebkitJustifyContent": "center",
            "alignItems": "stretch",
            "backgroundColor": "rgba(0,175,255,1.00)",
            "borderBottomColor": "rgba(0,175,255,1.00)",
            "borderBottomLeftRadius": "50px",
            "borderBottomRightRadius": "50px",
            "borderBottomStyle": "solid",
            "borderBottomWidth": "0px",
            "borderLeftColor": "rgba(0,175,255,1.00)",
            "borderLeftStyle": "solid",
            "borderLeftWidth": "0px",
            "borderRightColor": "rgba(0,175,255,1.00)",
            "borderRightStyle": "solid",
            "borderRightWidth": "0px",
            "borderTopColor": "rgba(0,175,255,1.00)",
            "borderTopLeftRadius": "50px",
            "borderTopRightRadius": "50px",
            "borderTopStyle": "solid",
            "borderTopWidth": "0px",
            "boxShadow": "0px 0.75px 1.5px rgba(0,0,0,0.24)",
            "display": "flex",
            "justifyContent": "center",
            "marginBottom": "0px",
            "marginLeft": "0px",
            "marginRight": "0px",
            "marginTop": "0px",
            "minHeight": "44px",
            "minWidth": "64px",
            "msFlexAlign": "stretch",
            "msFlexPack": "center",
            "paddingBottom": "0px",
            "paddingLeft": "0px",
            "paddingRight": "0px",
            "paddingTop": "0px",
            "width": "100%",
          }
        }
      >
        <div
          className="css-cursor-18t94o4 css-view-1dbjc4n"
          data-focusable={true}
          disabled={false}
          onKeyDown={[Function]}
          onKeyPress={[Function]}
          onKeyUp={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          role="button"
          style={
            Object {
              "borderBottomLeftRadius": "50px",
              "borderBottomRightRadius": "50px",
              "borderTopLeftRadius": "50px",
              "borderTopRightRadius": "50px",
              "cursor": "pointer",
              "msTouchAction": "manipulation",
              "overflowX": "hidden",
              "overflowY": "hidden",
              "position": "relative",
              "touchAction": "manipulation",
            }
          }
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "WebkitAlignItems": "center",
                "WebkitBoxAlign": "center",
                "WebkitBoxDirection": "normal",
                "WebkitBoxOrient": "horizontal",
                "WebkitBoxPack": "center",
                "WebkitFlexDirection": "row",
                "WebkitJustifyContent": "center",
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
                "msFlexAlign": "center",
                "msFlexDirection": "row",
                "msFlexPack": "center",
              }
            }
          >
            <div
              className="css-text-901oao css-textOneLine-bfa6kz"
              dir="auto"
              style={
                Object {
                  "color": "rgba(255,255,255,1.00)",
                  "direction": "ltr",
                  "fontFamily": "Roboto-Medium, Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                  "letterSpacing": "1px",
                  "marginBottom": "9px",
                  "marginLeft": "16px",
                  "marginRight": "16px",
                  "marginTop": "9px",
                  "textAlign": "center",
                }
              }
            >
              <div
                className="css-view-1dbjc4n"
                style={
                  Object {
                    "WebkitAlignItems": "center",
                    "WebkitBoxAlign": "center",
                    "WebkitBoxDirection": "normal",
                    "WebkitBoxFlex": 1,
                    "WebkitBoxOrient": "horizontal",
                    "WebkitFlexBasis": "0%",
                    "WebkitFlexDirection": "row",
                    "WebkitFlexGrow": 1,
                    "WebkitFlexShrink": 1,
                    "alignItems": "center",
                    "display": "flex",
                    "flexBasis": "0%",
                    "flexDirection": "row",
                    "flexGrow": 1,
                    "flexShrink": 1,
                    "msFlexAlign": "center",
                    "msFlexDirection": "row",
                    "msFlexNegative": 1,
                    "msFlexPositive": 1,
                    "msFlexPreferredSize": "0%",
                  }
                }
              >
                <span
                  className="css-text-901oao css-textHasAncestor-16my406"
                  dir="auto"
                  style={
                    Object {
                      "WebkitAlignItems": "center",
                      "WebkitBoxAlign": "center",
                      "WebkitBoxPack": "center",
                      "WebkitJustifyContent": "center",
                      "alignItems": "center",
                      "color": "rgba(255,255,255,1.00)",
                      "direction": "ltr",
                      "fontFamily": "Roboto",
                      "fontSize": "16px",
                      "fontWeight": "500",
                      "justifyContent": "center",
                      "letterSpacing": "0px",
                      "lineHeight": "22px",
                      "msFlexAlign": "center",
                      "msFlexPack": "center",
                      "paddingLeft": "5px",
                      "paddingRight": "5px",
                      "paddingTop": "1px",
                      "textAlign": "center",
                      "textDecoration": "none",
                      "textTransform": "uppercase",
                    }
                  }
                >
                  I UNDERSTAND
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
